metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -644,11 +644,27 @@ object MetalsEnrichments
         localOccurrence.symbol == symbol
       }
     }
+
+    /**
+     * Convert given symbol into location in code even though symbol's range may not exist.
+     */
     def definition(uri: String, symbol: String): Option[l.Location] = {
       textDocument.occurrences
         .find(o => o.role.isDefinition && o.symbol == symbol)
         .map { occ => occ.toLocation(uri) }
     }
+
+    /**
+     * Convert given symbol into location in code only if symbol's occurrence
+     * has range defined.
+     */
+    def toLocation(uri: String, symbol: String): Option[l.Location] =
```

Comment:
In reality the range should always exist, so no need to create a separate method.

Reply:
I never was a fan of `should always` and `should never happen` :/ In my use case, it's really ok to get the Option back, so I would leave it.

Reply:
> Did you faced such case?

I think it was caused by semanticDB with legacy schema, but if I no longer use them these safety cautions can be removed.